sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ResourceEvaluationStage.java

```diff
@@ -150,9 +151,10 @@ public EvaluationOutcome evaluate(MesosResourcePool mesosResourcePool, PodInfoBu
 
         return pass(
                 this,
-                offerRecommendation == null ? null : Arrays.asList(offerRecommendation),
-                "Offer contains sufficient '%s'",
-                resourceRequirement.getName());
+                offerRecommendation == null ? Collections.emptyList() : Arrays.asList(offerRecommendation),
+                "Offer contains sufficient '%s': requirement=%s",
+                resourceRequirement.getName(),
+                TextFormat.shortDebugString(resourceRequirement.getValue()));
```

Comment:
I wonder if there's a better way to include these -- obviously you want some record of the real protos being compared in the logs, but when they're mixed up in logging about what _happened_ it gets annoying to have to read around them. I wonder if we could structure things so that they're presented first, with some ad hoc ID (the way that offers are) and then referenced by that ID in actual evaluation logging.